homeassistant/components/mobile_app/webhook.py

```diff
@@ -382,8 +379,11 @@ async def webhook_register_sensor(hass, config_entry, data):
         _LOGGER.error("Error registering sensor: %s", ex)
         return empty_okay_response()
 
-    register_signal = f"{DOMAIN}_{data[ATTR_SENSOR_TYPE]}_register"
-    async_dispatcher_send(hass, register_signal, data)
+    if not existing_sensor:
+        register_signal = f"{DOMAIN}_{data[ATTR_SENSOR_TYPE]}_register"
+        async_dispatcher_send(hass, register_signal, data)
+    else:
+        async_dispatcher_send(hass, SIGNAL_SENSOR_UPDATE, data)
```

Comment:
Let's turn around `if not X:…; else:` into `if X: …; else:`

Reply:
I approached it from the angle: Handle the main thing first, else is a way out...

As this method is for registering, not for handling, the first part is the registering handling.

Reply:
Switched them.